### PR TITLE
add an integration spec for breakers

### PR DESCRIPTION
**Pull request by aub**
Add a spec that shows breakers turning on and off an outage in Rx.

### CODE REVIEW

File: spec/lib/rx/api/prescriptions_spec.rb [hunk 1]
```diff
@@ -102,4 +102,32 @@
...
+      stub_varx_request(:get, 'mhv-api/patient/v1/prescription/getactiverx', active_rxs, status_code: 500)
+      20.times do
+        begin
+          client.get_active_rxs
+        rescue Common::Client::Errors::ClientResponse
+          nil
+        end
+      end
+
+      expect { client.get_active_rxs }.to raise_exception(Breakers::OutageException)
```
Comment (saneshark) [inline spec/lib/rx/api/prescriptions_spec.rb]: So this is awesome, but will this also be a serializable error class? What will we want to return to the api consumer? Maybe an Internal Server is satisfactory, but we could possibly have a more detailed error message that gets rendered. Either one of Common::Exceptions or something new.

Comment (saneshark) [inline spec/lib/rx/api/prescriptions_spec.rb]: in any case, I think we would rescue this exception and raise something else that is serializable. This works!

Comment (aub) [inline spec/lib/rx/api/prescriptions_spec.rb]: @saneshark You make a fine point. I will rewrite this as a true integration test that goes through the controller and make sure the error handling is done correctly.

**Comment by aub** [comment]
@saneshark Ok, I'm now properly handling the Breakers::OutageException in ApplicationController and returning a 503 to the client. I've also changed the spec to be more of a proper integration test, using the controller to make the requests. Anything else I should look into?

**Comment by aub** [comment]
@saneshark If you have a chance to look at this that'd be great. I think I've integrated it properly with your error handling code but want to make sure.

**Comment by saneshark** [comment]
LGTM

**Pull request merged**